cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/controls-side-bar.tsx

```diff
@@ -213,24 +220,29 @@ export default function ControlsSideBarComponent(props: Props): JSX.Element {
             <ObservedDrawRectangleControl
                 canvasInstance={canvasInstance}
                 isDrawing={activeControl === ActiveControl.DRAW_RECTANGLE}
+                disabled={!labels.length}
```

Comment:
Instead of adding the property "disabled" to each element what do you think about adding such css

```css
{
    opacity: 0.5;
    pointer-events: none;
}
```

to Layout.Sider?

Reply:
It would allow us to get rid of a lot of changes

Reply:
Okay, got it. In this case we lose ability to fit, zoom, move, etc.